bin/schemas/story.json

```diff
@@ -0,0 +1,846 @@
+              "default": "",
+              "title": "Date the media file was uploaded",
+              "format": "date-time"
+            },
+            "posterId": {
+              "type": "integer",
+              "default": 0,
+              "title": "ID of the resource's poster"
+            }
+          }
+        },
+        "overlay": {
+          "type": "object",
+          "$ref": "#/$defs/linearPattern"
+        }
+      }
+    },
+    "shapeElement": {
+      "type": "object",
+      "title": "Shape element",
+      "allOf": [
+        {
+          "$ref": "#/$defs/element"
+        }
+      ]
+    },
+    "pattern": {
+      "type": "object",
+      "oneOf": [
+        { "$ref": "#/$defs/solidPattern" },
+        { "$ref": "#/$defs/linearPattern" }
+      ]
+    },
+    "solidPattern": {
+      "type": "object",
+      "title": "Solid pattern",
+      "required": ["color"],
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["none", "solid", "linear", "radial"]
+        },
+        "color": {
+          "type": "object",
+          "title": "RGB(A) color",
+          "required": ["r", "g", "b"],
+          "properties": {
+            "r": {
+              "type": "integer",
+              "title": "Red",
+              "minimum": 0,
+              "maximum": 255
+            },
+            "g": {
+              "type": "integer",
+              "title": "Green",
+              "minimum": 0,
+              "maximum": 255
+            },
+            "b": {
+              "type": "integer",
+              "title": "Blue",
+              "minimum": 0,
+              "maximum": 255
+            },
+            "a": {
+              "type": "number",
+              "title": "Alpha",
+              "minimum": 0,
+              "maximum": 1
+            }
+          }
+        }
+      }
+    },
+    "linearPattern": {
+      "type": "object",
+      "title": "Linear color pattern",
+      "required": ["type", "stops"],
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["none", "solid", "linear", "radial"]
+        },
+        "stops": {
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "type": "object",
+            "allOf": [
+              {
+                "$ref": "#/$defs/solidPattern"
+              }
+            ],
+            "properties": {
+              "position": {
+                "type": "number",
+                "minimum": 0,
+                "maximum": 1
+              }
+            }
+          }
+        },
+        "alpha": {
+          "type": "number",
+          "minimum": 0,
+          "maximum": 1
+        },
+        "rotation": {
+          "type": "number",
+          "minimum": -360,
+          "maximum": 360
+        }
+      }
+    },
+    "pageTemplateType": {
+      "type": "string",
+      "title": "The pageTemplateType Schema",
+      "enum": [
+        "",
```

Comment:
Hm, is `""` for the custom page templates?

Reply:
No, for the default templates. The template pages that shouldn't be listed as page templates currently have empty strings as the `pageTemplateType`. There's also a Jest test that enforcing that all templates have this property. See #5889 and discussion on that PR.

I think we can change this from `""` to `null` though.

Reply:
Aside: while we need to store `pageTemplateType` for the default templates, there's no need to retain that when actually inserting the template in the editor.

Reply:
Removing from schema now.